### DISCUSSION TITLE
My minds got a mind of it's own

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27976] [Discussions]
Anyone else been a pain in the ass like me because of confusion and inexperience? I keep flip flopping how I feel and think about all this. I have become a stronger, clearer and more confident person through my travels in this Universe but I keep slipping up and causing drama. Obviously the mind the culprit. In a lot of ways I am grateful and because I was once blind and now I can see I am elevated to new sense of being. On the flip side I keep cating like an immature child who just caught his girlfriend kissing his friend. I can't seem to get over being illegally invaded in my privacy. But then. Agod friend of mine from another country asked me last weekend why privacy is so important to me. It turned out to be. A good question and I couldn't really answer it except because it was my right. I felt stupid.
How do I get over these things holding me back and my fear that tis is all  a plan to frame me or something. I don't know. What I do know is that I should really be grateful and use my new found wisdom and vision to move forward and try living more in the present.  If anyone is familiar with me and my adventures in the verse and beyond can you please tell me the exact amount of damage I may have caused besides the obvious? Am I way past redemption? Have I ruined whatever possibilities or relationships that I my have had if I handled all this different?

**Comment by Adcrane966** [comment]
> Anyone else been a pain in the ass like me because of confusion and inexperience? I keep flip flopping how I feel and think about all this. I have become a stronger, clearer and more confident person through my travels in this Universe but I keep slipping up and causing drama. Obviously the mind the culprit. In a lot of ways I am grateful and because I was once blind and now I can see I am elevated to new sense of being. On the flip side I keep cating like an immature child who just caught his girlfriend kissing his friend. I can't seem to get over being illegally invaded in my privacy. But then. Agod friend of mine from another country asked me last weekend why privacy is so important to me. It turned out to be. A good question and I couldn't really answer it except because it was my right. I felt stupid. How do I get over these things holding me back and my fear that tis is all a plan to frame me or something. I don't know. What I do know is that I should really be grateful and use my new found wisdom and vision to move forward and try living more in the present. If anyone is familiar with me and my adventures in the verse and beyond can you please tell me the exact amount of damage I may have caused besides the obvious? Am I way past redemption? Have I ruined whatever possibilities or relationships that I my have had if I handled all this different?

**Reply by ghost** [reply]
I just really feel alone in all this. I feel a little better now that I discovered some answers but I had to find it all out by myself. For a long time I was in survival mode. Like all I knew how to do was put up a defensive wall. I thought I was being attacked. Now that I have come so far I am much clearer and stronger. You have all helped me with this but I keep slipping back here and there. 
How do I keep myself empowered? How do I stay strong and awake to my present? Why do I keep acting like a child who just cannot get his way? And the funny thing is you all allow me to get my way. You've taught me that I am just fine with who I am and that I shouldn't put so much guilt and negative perception on everything. 
Also why do I keep allowing my imagination to fill in the blanks mad the answers that I don't have any do not necessarily need to know?
I have come to the conclusion that I need all of you just as much as you need me. I truly believe we were brought together for a reason. 
All I ask is for you to accept me for who I am and help me in any way possible to keep going and learning and keeping my eyes open while not judging or creating my own perceptions to what my eyes see. I know I make it difficult to help me. I'm not easy I'll admit that. But everyone knows that I have a warm heart, empathy for all of you and humanity. I may be a bit frisky, but so are all of you hehe..and I like that. You tell me all the time not to feel guilty because of this. I thank you for that. As an American and born Christian it kind of comes with my conditioning lol
All and all though, I feel and see a big difference in me from where I was last year. I don't rant and rave anymore haha.. although a lot of people found my words to be impactful..except that sometimes I didn't know when to stop. 
On that note.....I can't say how much I ma grateful ... although I may not show it. 
Thank you Thank you Thank you
